doc/data_protocol.md

```diff
@@ -342,6 +463,8 @@ The source file can be found in [doc/diagrams/telemetry-sequence.xml](diagrams/t
 
 ### Data description
 
+ALL DATA MUST BE LITTLE-ENDIAN
+
 #### Engine Controller
```

Comment:
While I appreciate that this data as a snapshot of what it looks like in the https://github.com/aesirkth/edda-engine-electronics-software/tree/codegen branch has been put in a table, I cannot commit to this section at this time.

Everything is still in a phase of exploration, so putting something that is still a borderline experiment in documentation would to me be wrong.

I expect that the underlying protocol will change quite rapidly and often as the development of the software for Edda starts properly 😄

Reply:
Yes, I am aware, it was an attempt to have a more solid idea on how the data sent from the EC would look like so we can continue working on the engine dashboard. Maybe we could mark this as temporary/experimental or simply left it on the development branch until we have the final one.